src/main/java/com/example/CourseWork2/service/ExaminerServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.example.CourseWork2.service;
+
+import com.example.CourseWork2.entity.Question;
+import com.example.CourseWork2.excrption.IncorrectAmountOfQuestionsException;
+import org.springframework.stereotype.Service;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+
+@Service
+public class ExaminerServiceImpl implements ExaminerService{
+    private Random random;
+    private JavaQuestionService javaQuestionService;
```

Comment:
```suggestion
    private QuestionService questionService;
```
инжектить надо по интерфейсу всегда